private/pkg/diff/diffmyers/diffmyers.go

```diff
@@ -51,6 +53,107 @@ func Diff(from, to [][]byte) []Edit {
 	return shortestEdits(from, to, 0, 0)
 }
 
+// Print prints the edits in the unified diff format without the header.
+//
+// Ref: https://www.gnu.org/software/diffutils/manual/html_node/Detailed-Unified.html
+func Print(from, to [][]byte, edits []Edit) ([]byte, error) {
+	const contextThreshold = 2
```

Comment:
not a huge deal, but it seems like the context for hunks in `diff` (at least, macOS/BSD `diff`) is 3 (and seems the same on GitHub, at least looking at the context given between hunks in this PR). Is there a reason we chose 2?

Reply:
It does 3 based on the logic that is when the threshold crossed 2 which is 3 we stop. But you are right this is a bit confusing we can update this to match 3 here.